win32/gpredict.exe.manifest

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='UTF-8' standalone='yes'?>
 <assembly xmlns='urn:schemas-microsoft-com:asm.v1' manifestVersion='1.0'>
-  <assemblyIdentity name="GPredict" processorArchitecture="x86" version="1.3.99.0" type="win32"/>
+  <assemblyIdentity name="GPredict" processorArchitecture="x86" version="VERSION_STRL" type="win32"/>
```

Comment:
Are we generating 32 bit binaries with mingw64 or does this need to be updated?

Reply:
We are generating 32-bit binaries (-m32 in CC options). In the interests of avoiding complexity for the user (ie: deciding on a download!) I'd like to stick with 32-bit until we have a proven need for 64-bit on Windows.

Reply:
32 bit is fine, I was just wondering whether mingw64 implied 64 bit binaries.